nestor_api/lib/config.py

```diff
@@ -46,6 +46,16 @@ def get_app_config(app_name: str, config_path: str = Configuration.get_config_pa
     return _resolve_variables_deep(config)
 
 
+def get_applications(app_config: dict) -> list:
+    """Get applications processes (not cronjobs)"""
+    return list(filter(lambda process: process["is_cronjob"] is False, app_config["processes"]))
```

Comment:
What about comprehension here like: 
```python
return [process for process in app_config["processes"] if process["is_cronjob"] is False]
```
I find it clearer to read than function nesting with lambda